conjure-java-undertow-runtime/src/test/java/com/palantir/conjure/java/undertow/runtime/AuthTest.java

```diff
@@ -55,7 +64,35 @@ public void testAuthHeaderMultipleValues() {
         exchange.getRequestHeaders().add(Headers.AUTHORIZATION, "Bearer foo");
         exchange.getRequestHeaders().add(Headers.AUTHORIZATION, "Bearer bar");
         assertThatThrownBy(() -> CONTEXT.auth().header(exchange))
-                .isInstanceOf(SafeIllegalArgumentException.class)
-                .hasMessage("One Authorization header value is required");
+                .isInstanceOf(ServiceException.class)
+                .hasMessageContaining("ServiceException: UNAUTHORIZED (Conjure:MalformedCredential)");
+    }
+
+    //delete
```

Comment:
delete?

Reply:
doh, I copy/pasted the tests to rename and add coverage for cookie auth. Marked was to differentiate methods with duplicate names while I renamed them. Update incoming.